lms/models/grouping.py

```diff
@@ -104,9 +106,23 @@ class CanvasGroup(Grouping):
     __mapper_args__ = {"polymorphic_identity": Grouping.Type.CANVAS_GROUP}
 
 
-class BlackBoardGroup(Grouping):
+class BlackboardGroup(Grouping):
     __mapper_args__ = {"polymorphic_identity": Grouping.Type.BLACKBOARD_GROUP}
 
 
 class Course(Grouping):
     __mapper_args__ = {"polymorphic_identity": Grouping.Type.COURSE}
+
+
+class GroupingMembership(CreatedUpdatedMixin, BASE):
+    __tablename__ = "grouping_membership"
```

Comment:
I think you mentioned something about the table name @seanh

No strong opinions from my side on this "grouping_membership" works, "grouping_users" or something similar works as well.